### PR TITLE
Initial commit for Dashboard

### PR DESCRIPTION
**Pull request by sorja**
# First 
## part of #1333 
image

- [ ] CarbonGrowingStock
- [x] ForestArea
- [x] ForestAreaPercent
- [x] ForestAreaWithinProtectedAreas
- [x] ForestOwnership
- [x] hooks (data fetching)
- [x] NaturallyRegeneratingForest
- [x] PrimaryDesignatedManagementObjective
- [x] PrimaryForest
- [x] utils (chart preferences and options)

### CODE REVIEW

File: client/components/Chart/utils/ChartColors.ts [hunk 1]
```diff
@@ -0,0 +1,20 @@
+export enum ChartColors {
```
Comment (sorja) [inline client/components/Chart/utils/ChartColors.ts:1]: Todo: this file will be removed

File: client/components/Chart/utils/getChartOptions.ts [hunk 2]
```diff
@@ -0,0 +1,124 @@
+import { Numbers } from '@core/utils/numbers'
```
Comment (sorja) [inline client/components/Chart/utils/getChartOptions.ts:1]: This file will be removed

**Comment by sorja** [comment]
market as draft until extra files removed and some code cleaned

File: client/pages/AssessmentHome/FraHome/FraHome.tsx [hunk 3]
```diff
@@ -54,21 +55,24 @@ const FraHome: React.FC = () => {
...
-          from={BasePaths.Assessment.home()}
-          to={BasePaths.Assessment.home(countryIso, assessmentName, cycleName, AssessmentHomeRouteNames.overview)}
-          exact
-        />
-        {sections.map(({ name, component }) => (
-          <Route
-            key={name}
-            path={BasePaths.Assessment.home(countryIso, assessmentName, cycleName, name)}
-            component={component}
+      {displayTabs ? (
```
Comment (minotogna) [inline client/pages/AssessmentHome/FraHome/FraHome.tsx:58]: it was already handled. now there's no need to add this. Dashboard is always added. leave it as it was

File: client/pages/Dashboard/CarbonGrowingStock/CarbonGrowingStock.tsx [hunk 4]
```diff
@@ -0,0 +1,32 @@
...
+  const isIsoCountry = Areas.isISOCountry(countryIso)
+  const columns = ['rowName', '1990', '2000', '2010', '2020']
+  const variables = ['growingStockTotal', 'carbonStockBiomassTotal', 'carbonStockTotal']
+  const units = isIsoCountry
+    ? ['millionsCubicMeterOverBark', 'tonnesPerHa', 'tonnesPerHa']
+    : ['billionCubicMeter', 'gt', 'gt']
+  const tableNames = ['carbonAndGrowingStock']
+  const section = 'carbonAndGrowingStock'
+
+  // TODO Missing data
```
Comment (minotogna) [inline client/pages/Dashboard/CarbonGrowingStock/CarbonGrowingStock.tsx:22]: can u specify what data is missing ?

File: client/pages/Dashboard/components/table/table.tsx [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+
+type Props = {
+  columns: any[]
+  variables: any[]
+  units: any[]
+  tableNames: string[]
+  section: string
+}
+
+const Table = (props: Props) => {
```
Comment (minotogna) [inline client/pages/Dashboard/components/table/table.tsx:21]: file name and folder name upper case

File: i18n/resources/en/statisticalFactsheets.js [hunk 6]
```diff
@@ -1,17 +1,17 @@
...
+    naturalForestArea: 'Natural regenerating forest',
+    plantedForest: '$t(forestCharacteristics.plantedForest)',
+    forestAreaWithinProtectedAreas: '$t(forestAreaWithinProtectedAreas.header)',
+    landArea: 'Land area',
+    primaryForest: '$t(specificForestCategories.primaryForest)',
+    privateOwnership: '$t(forestOwnership.privateOwnership)',
+    publicOwnership: '$t(forestOwnership.publicOwnership)',
+    otherArea: 'Other area',
+    otherForest: 'Other forest',
+    otherOrUnknown: 'Other or unknown',
```
Comment (minotogna) [inline i18n/resources/en/statisticalFactsheets.js:14]: should other languages be updated as well ?

**Review by minotogna — CHANGES_REQUESTED** [review]

File: client/pages/AssessmentHome/FraHome/FraHome.tsx (hunk 3 above)
Comment (minotogna) [reply client/pages/AssessmentHome/FraHome/FraHome.tsx:58]: check useSections

File: client/pages/Dashboard/components/table/table.tsx [hunk 7]
```diff
@@ -0,0 +1,88 @@
...
+                <tr key={variable}>
+                  {['', ...columns].map((column: any, i: any) => {
+                    if (i === 0)
+                      return (
+                        <th key={`${variable}-${column}`} className="fra-table__category-cell">
+                          {`${t(variable)} (${i18n.t(`unit.${units[rowIdx]}`)})`}
+                        </th>
+                      )
+                    // const row = data.find((entry: any) => entry.rowName === tableRow) || {}
+                    const nodeValue = tableData[countryIso][tableNames[0]][column]?.[variable]
```
[the remark this answers is not in the document]
Comment (sorja) [reply client/pages/Dashboard/components/table/table.tsx:70]: In this case we have exact values for tableName, columnName and variableName; tableDatas helper wants Type Table, Col, Row so I would need to create objects, I think this is readable enough like this for this single case

**Review by minotogna — APPROVED** [review]

**Pull request merged**